dirac.cfg

```diff
@@ -748,6 +748,35 @@ Operations
         }
       }
     }
+    # Specify how job access their data
+    # See https://dirac.readthedocs.io/en/latest/AdministratorGuide/Systems/WorkloadManagement/InputDataResolution.html
+    InputDataPolicy
```

Comment:
Is anything inside here mandatory? Or everything is optional? Can you specify?

Reply:
done